src/pages/AddProduct.tsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from "react";
+import { addProduct } from "../api/products";
+
+const AddProduct = () => {
+  const [product, setProduct] = useState({
```

Comment:
Nitpick: Maybe it would be more readable if you create a `emptyProduct` variable (outside of the function) with this object in it and assign it as the default value.